cli/cli/infra.py

```diff
@@ -52,156 +37,119 @@
 DB_INSTANCE_NAME = "scw-sls-gw"
 DB_ENGINE = "PostgreSQL-14"
 DB_USERNAME = "kong"
-DB_PASSWORD = "K0ngkongkong!"
 DB_VOLUME_TYPE = "lssd"
 DB_NODE_TYPE = "DB-DEV-S"
-DB_VOLUME_SIZE = "5000000000"  # Expressed in bytes
+DB_VOLUME_SIZE = 5000000000  # Expressed in bytes
 
 # Name is fixed for Scaleway managed database
 DB_DATABASE_NAME = "rdb"
-DB_DATABASE_NAME_LOCAL = "kong"
 
 
-class InfraManager(object):
-    def __init__(self):
+class InfraManager:
+    """Manager for the infrastructure of the gateway."""
```

Comment:
I think this class still needs another round of refactoring. My main pet peeve is that it's too big, and doesn't have enough responsibility to be a class and not a collection of functions. Its only attribute is the apis instance, and they could be passed as an argument to each function.